router/logic_test.go

```diff
@@ -31,7 +31,7 @@ func verifyRoute(t *testing.T, rs *routeSet, inUrl string, expectOutUrl string)
 	}
 }
 
-func TestRouteMatching(t *testing.T) {
+func TestUnitRouteMatching(t *testing.T) {
```

Comment:
I use TestUnit so that i can only run unit tests without running the Integration test, for example: go test router -run TestUnit.